### PR TITLE
Remove attributes from loadCSS polyfill inline script

### PR DESCRIPTION
**Pull request by greggraf**
Update tests and code to remove attributes from loadCSS polyfill inline script

Inline script to polyfill link rel=preload behavior in browsers like Safari was not getting processed. After removing the `nomodule` and `integrity` attributes it is processed and `bundle.css` is loaded.

This work is to solve #346

### CODE REVIEW

**Comment by goto-bus-stop** [comment]
thanks! does it work when you only remove the `integrity` attribute but keep the `nomodule` one? or do they both need to go? afaik `integrity` does nothing for inline scripts, but `nomodule` should still prevent that code from executing in new browsers.

File: test/document.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ tape('server render choo apps', function (assert) {
...
...ner("load",f),i.media=d||"all"}var g,h=a.document,i=h.createElement("link");if(c)g=c;else{var j=(h.body||h.getElementsByTagName("head")[0]).childNodes;g=j[j.length-1]}var k=h.styleSheets;i.rel="stylesheet",i.href=b,i.media="only x",e(function(){g.parentNode.insertBefore(i,c?g:g.nextSibling)});var l=function(a){for(var b=i.href,c=k.length;c--;)if(k[c].href===b)return a();setTimeout(function(){l(a)})};return i.addEventListener&&i.addEventListener("load",f),i.onloadcssdefined=l,l(f),i};"undefined"!=typeof exports?exports.loadCSS=b:a.loadCSS=b})("undefined"!=typeof global?global:this);;(function(a){if(a.loadCSS){var b=loadCSS.relpreload={};if(b.support=function(){try{return a.document.createElement("link").relList.supports("preload")}catch(b){return!1}},b.poly=function(){for(var b=a.document.getElementsByTagName("link"),c=0;c<b.length;c++){var d=b[c];"preload"===d.rel&&"style"===d.getAttribute("as")&&(a.loadCSS(d.href,d,d.getAttribute("media")),d.rel=null)}},!b.support()){b.poly();var c=a.setInterval(b.poly,300);a.addEventListener&&a.addEventListener("load",function(){b.poly(),a.clearInterval(c)}),a.attachEvent&&a.attachEvent("onload",function(){a.clearInterval(c)})}}})(this);</script>
```
Comment (goto-bus-stop) [inline test/document.js]: we can also get rid of the `__PRELOAD_INTEGRITY__` variable and replaces in this file

Comment (greggraf) [reply test/document.js]: In my testing, the `nomodule` attribute prevents the inline script from running in both Safari and Chrome. This behavior appears correct for when ES6 Modules (and the `nomodule` attribute) are supported.

For versions of Safari that support ES6 modules but not the `rel="preload"` resource hint, the script should run regardless of module support to polyfill the lack of preload support, so the `nomodule` attribute should not be added back.

**Comment by greggraf** [comment]
In my testing, the `nomodule` attribute prevents the inline script from running in both Safari and Chrome. This behavior appears correct for when ES6 Modules (and the `nomodule` attribute) are supported.

For versions of Safari that support ES6 modules but not the `rel="preload"` resource hint, the script should run regardless of module support to polyfill the lack of preload support, so the `nomodule` attribute should not be added back.

**Comment by goto-bus-stop** [comment]
> For versions of Safari that support ES6 modules but not the `rel="preload"` resource hint

gotcha, makes sense--this is the context i was missing :) thanks for explaining!

this LGTM then, if you could remove the remaining references to `__PRELOAD_INTEGRITY__` in the test file?

File: test/document.js (hunk 1 above)
Comment (greggraf) [reply test/document.js]: Right! Done!

**Comment by goto-bus-stop** [comment]
thanks! 🎉

**Comment by yoshuawuyts** [comment]
Oh, by the way - I found out the other day how to do CSP for inline scripts :D https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src

**Pull request merged**